src/components/Run.js

```diff
@@ -31,19 +32,12 @@ Run.propTypes = {
 };
 
 export default connect(
-  function mapStateToProps(state) {
-    return {
-      running: state.running
-    };
-  },
-  function mapDispatchToProps(dispatch) {
-    return {
-      onRun: function() {
-        dispatch(run());
-      },
-      onStop: function() {
-        dispatch(stop());
-      }
-    };
-  }
+  (state) => ({
+    running: state.running
+  }),
+  (dispatch) => {
+    return bindActionCreators( {
```

Comment:
you could just do 

```javascript
dispatch => bindActionCreators(...)
```
since you don't need curly braces if your function is just one line.
```